lms/djangoapps/bulk_email/tests/test_email.py

```diff
@@ -210,7 +210,7 @@ def test_send_to_all(self):
 
         test_email = {
             'action': 'Send email',
-            'send_to': 'all',
+            'send_to': '["myself", "staff", "learners"]',
```

Comment:
It's not intending "all except cohort", just "all users", which requires selecting all options (that are currently enabled)